torchvision/models/efficientnet.py

```diff
@@ -32,15 +32,23 @@
 
 
 class SqueezeExcitation(nn.Module):
```

Comment:
I'm still on the fence on whether we should mark this class with `_` to explicitly denote it's private (I believe that's what you did on your other PR) or leave as is for when we move it to `ops`.  Thoughts?

Would you consider making this PR replace ALL existing SE layers? If yes, we could move this straight to `ops`. Happy to help with converting existing models in a BC manner.

Reply:
> I'm still on the fence on whether we should mark this class with _ to explicitly denote it's private
I'm considering to make it public since it has been published for a while and still popular (https://paperswithcode.com/method/squeeze-and-excitation-block)

 > Would you consider making this PR replace ALL existing SE layers? If yes, we could move this straight to ops. Happy to help with converting existing models in a BC manner.
I would like to get the consensus in #4333 first and execute carefully. I'm thinking to merge this PR first to avoid the third SE implementation and then consolidate the two into one.